### ISSUE TITLE
Some Flint static files don't exist

### ISSUE THREAD

**Issue by JoelEinbinder** [flintjs/flint#110]
After I make a flint app and try to view it in my browser, it tries to load a bunch of static files like /__/react.dev.js But these return html pages (the default page for react.dev.js.

![screen shot 2015-11-08 at 2 25 15 am](https://cloud.githubusercontent.com/assets/4624233/11019479/ff42b324-85bf-11e5-947d-650be97658f4.png)

I'm using node v5 which somebody on slack seemed to point to as the issue. Heres my output for flint --debug. I am on osx yosemite and installed flint with npm install -g

```
~/flint/app (master ✘)✭ ᐅ flint --debug

opts { debug: true,
  port: undefined,
  host: undefined,
  watch: undefined,
  hasRunInitialBuild: false,
  build: undefined,
  defaultPort: 4000,
  modulesDir: '/usr/local/lib/node_modules/flint/node_modules/flint-runner/node_modules',
  appDir: '/Users/joel/flint/app',
  dir: '/Users/joel/flint/app',
  flintDir: '/Users/joel/flint/app/.flint',
  internalDir: '/Users/joel/flint/app/.flint/.internal',
  depsDir: '/Users/joel/flint/app/.flint/.internal/deps',
  template: '.flint/index.html',
  buildDir: '/Users/joel/flint/app/.flint/build',
  deps: 
   { dir: '/Users/joel/flint/app/.flint/.internal/deps',
     internalsIn: '/Users/joel/flint/app/.flint/.internal/deps/internals.in.js',
     internalsOut: '/Users/joel/flint/app/.flint/.internal/deps/internals.js',
     depsJS: '/Users/joel/flint/app/.flint/.internal/deps/deps.js',
     depsJSON: '/Users/joel/flint/app/.flint/.internal/deps/deps.json',
     packagesJS: '/Users/joel/flint/app/.flint/.internal/deps/packages.js' },
  configFile: '/Users/joel/flint/app/.flint/.internal/flint.json',
  outDir: '/Users/joel/flint/app/.flint/.internal/out',
  name: 'app',
  saneName: 'app',
  url: 'app.dev' }
opts.get deps { dir: '/Users/joel/flint/app/.flint/.internal/deps',
  internalsIn: '/Users/joel/flint/app/.flint/.internal/deps/internals.in.js',
  internalsOut: '/Users/joel/flint/app/.flint/.internal/deps/internals.js',
  depsJS: '/Users/joel/flint/app/.flint/.internal/deps/deps.js',
  depsJSON: '/Users/joel/flint/app/.flint/.internal/deps/deps.json',
  packagesJS: '/Users/joel/flint/app/.flint/.internal/deps/packages.js' }
cache: baseDir /Users/joel/flint/app
opts.get configFile /Users/joel/flint/app/.flint/.internal/flint.json
opts.get name app
Running app

outDir
opts.get outDir /Users/joel/flint/app/.flint/.internal/out
opts.get undefined undefined
opts.set port 4000
opts.set host localhost
opts.get port 4000
opts.get undefined undefined
opts.get hasRunInitialBuild false
opts.get hasRunInitialBuild false
cache: setViews { 'main.js': { views: [ 'Main' ] } }
opts.get hasRunInitialBuild false
opts.get hasRunInitialBuild false
opts.get appDir /Users/joel/flint/app
 ✓ main.js - 90ms
build took  90 ms
is new file true
opts.get hasRunInitialBuild false
scanFile /Users/joel/flint/app/main.js
checkInternals /Users/joel/flint/app/main.js
checkInternals: found false already undefined
installExternals /Users/joel/flint/app/main.js
cache: setFileImports /Users/joel/flint/app/main.js []
installing... []
opts.get deps { dir: '/Users/joel/flint/app/.flint/.internal/deps',
  internalsIn: '/Users/joel/flint/app/.flint/.internal/deps/internals.in.js',
  internalsOut: '/Users/joel/flint/app/.flint/.internal/deps/internals.js',
  depsJS: '/Users/joel/flint/app/.flint/.internal/deps/deps.js',
  depsJSON: '/Users/joel/flint/app/.flint/.internal/deps/deps.json',
  packagesJS: '/Users/joel/flint/app/.flint/.internal/deps/packages.js' }
installAll() fresh =  []
opts.set hasRunInitialInstall true
opts.set hasRunInitialBuild true
opts.get host localhost
opts.get port 4000

Ready ⇢ localhost:4000

opts.get undefined undefined
Shortcuts:

 • Open         • Verbose
                • Install (npm)

opts.get undefined undefined
opts.get configFile /Users/joel/flint/app/.flint/.internal/flint.json
readInstalled() []
cache: getImports:  []
npm: uninstall() toUninstall []
opts.get port 4000
```

And I deleted the app folder and remade. Heres the output for that
```
~/flint  ᐅ flint new app --debug

makeFolder /Users/joel/flint/app
 $  curl http://github.com
  ◠ Creating app...  
Online
mkdirp err null
Looking for updated scaffold in https://github.com/flintjs/scaffold
Check for new scaffold SHA in... /Users/joel/.flint/scaffoldSHA
Got  a4a10c9ea29527ecca6c7b242017fcb36dbc8eea
  ◜ Creating app...  Latest is a4a10c9ea29527ecca6c7b242017fcb36dbc8eea
checkNewScaffold needsNew? false
doCopy? true
Copy new scaffold /Users/joel/.flint/scaffold /Users/joel/flint/app
CONTINUE FROM getScaffold
Init new git
 $  git init
Initialized empty Git repository in /Users/joel/flint/app/.git/

Updating app name to app in /Users/joel/flint/app
Your Flint app is in ./app
```

I hope that something here was helpful.

**Comment by AndrewSB** [comment]
This bug exists on `node 4.2.2` as well

**Comment by natew** [comment]
@AndrewSB @JoelEinbinder checking this out now, finally got my VM upgraded to Capitan and it's not dog slow

**Comment by natew** [comment]
Ok turned out a dependency of runner was failing, and silently exiting the rest of the install.

I had to force delete my old flint folder. `rm -r /usr/local/lib/node_modules/flint`

And then reinstall `npm install -g flint`

Let me know if that fixes it!

**Comment by natew** [comment]
Still broken on 5.0, will check into it tomorrow.

**Comment by natew** [comment]
Fixed for 5.0

**Comment by AndrewSB** [comment]
:smile:
